types/bash-glob/bash-glob-tests.ts

```diff
@@ -0,0 +1,21 @@
+import * as bashGlob from 'bash-glob';
```

Comment:
please use `import bashGlob = require('bash-glob')` for commonjs modules, which use `export = bashGlob` syntax.

Reply:
fixed